spec/factories/participant_declaration.rb

```diff
@@ -8,20 +8,23 @@
     declaration_type { "started" }
 
     factory :ect_participant_declaration do
+      user { create(:user, :early_career_teacher) }
```

Comment:
Should we be creating users like this as part of the declaration?

Reply:
I don't understand your comment. What do you mean 'as part of the declaration'?

Reply:
In any case, I wonder if https://dfedigital.atlassian.net/browse/CPDTP-391 is not the better ticket to do it under.

Reply:
Oh, I think I get it. Do you mean "why does participant declaration create a user?"? 

The answer to that is "because without a user participant declaration makes no sense". And the extension is "without ect user, ect declaration makes no sense".